### ISSUE TITLE
Jump Jack lvl2 seems to desire both Reagent and Fluorine at the same time.

### ISSUE THREAD

**Issue by Chrisinnater5** [TheKilltech/rift_balance#2]
Description says it can take either, but if you plug in one resource it then requests the other. Haven't tested lvl3 yet.

**Comment by TheKilltech** [comment]
Indeed, thank you. Issue is that fluorine can be both input and output and that confuses the game. will try to get it fixed soon.

**Comment by Chrisinnater5** [comment]
Np, I'm absolutely loving this as an overhaul so far, and after seeing you didn't have a workshop link for comments, i figured your git would probably be a good place. I'll probably drop quite a few things in here ^_^.

**Comment by TheKilltech** [comment]
Thanks :), appreciate the feedback. Here or discord Riftbreaker server is a good place.

**Comment by TheKilltech** [comment]
should be fixed by now, except maybe for quite old save files which have the old pumpack building blueprint
